File: Selenium/Activities/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/drag-drop")
    print(f"the title of the page is: {driver.title}")
    footBall = driver.find_element(By.ID,"ball")
    dropzone1 = driver.find_element(By.ID,"dropzone1")
    dropzone2 = driver.find_element(By.ID,"dropzone2")
    actions = webdriver.ActionChains(driver)
    actions.drag_and_drop(footBall,dropzone1).pause(5).perform()
    if(driver.find_element(By.CLASS_NAME,"dropzone-text").text=="Dropped!"):
        print("The ball was dropped in dropzone1")

    actions.drag_and_drop(footBall,dropzone2).pause(5).perform()
    if(driver.find_element(By.CLASS_NAME,"dropzone-text").text=="Dropped!"):
        print("The ball was dropped in dropzone2")
    
    driver.quit()

File: Selenium/Activities/Activity10.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10 {
	public static void main(String args[]) {
		WebDriver driver = new FirefoxDriver();
		Actions builder = new Actions(driver);
		driver.get("https://training-support.net/webelements/drag-drop");
		System.out.println("The title of the page is: "+driver.getTitle());
		
		WebElement footBall = driver.findElement(By.id("ball"));
		WebElement dropzone1 = driver.findElement(By.id("dropzone1"));
		WebElement dropzone2 = driver.findElement(By.id("dropzone2"));
		
		//builder.clickAndHold(footBall).moveToElement(dropzone1).pause(3000).release().build().perform();
		builder.dragAndDrop(footBall,dropzone1).pause(3000).build().perform();
		if(driver.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
			System.out.println("The ball was dropped in dropzone1");
		}
		
		builder.dragAndDrop(footBall,dropzone2).pause(3000).build().perform();
		if(driver.findElement(By.className("dropzone-text")).getText().equals("Dropped!")) {
			System.out.println("The ball was dropped in dropzone2");
		}
		
		driver.quit();
			
		}
	}


